src/interaction/VSpherePair.hpp

```diff
@@ -80,8 +80,8 @@ namespace espressopp {
       virtual ~VSpherePair() {};
 
       void preset() {
-      	mfh = -(3.0/2.0);
-    	mth = -(5.0/2.0);
+      	mth = -(3.0/2.0);
+    	mfh = -(5.0/2.0);
         ef1 = epsilon*pow(1.0L*(2*M_PIl/3.0), 1.0L*mth);
```

Comment:
What is the point to ask for `long double` in the power and multiply it by `real`?